### PR TITLE
Client api version iss 1619

### PR DESCRIPTION
**Pull request by Wyvinar**

### CODE REVIEW

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-cli-pr-197-fhxtq for details. If you are unable to do so, please contact a member of the Galasa team.

File: pkg/runs/runsDownload.go [hunk 1]
```diff
@@ -227,20 +229,32 @@ func GetArtifactPathsFromRestApi(runId string, apiClient *galasaapi.APIClient) (
...
-	var artifactsList []galasaapi.ArtifactIndexEntry
-	artifactsList, httpResponse, err := apiClient.ResultArchiveStoreAPIApi.
-		GetRasRunArtifactList(context.Background(), runId).
-		Execute()
+	var restApiVersion string
+	restApiVersion, err = embedded.GetGalasactlRestApiVersion()
 
 	if err != nil {
-		err = galasaErrors.NewGalasaError(galasaErrors.GALASA_ERROR_RETRIEVING_ARTIFACTS_FAILED, err.Error())
+		log.Printf("Unable to retrieve galasactl rest api version.")
```
Comment (KirbyKatcher) [inline pkg/runs/runsDownload.go:236]: Shouldn't we have this as part of the parameter checker and just check if err !=nil before continuing. Lots of repetition of the same code here

**Review by KirbyKatcher — CHANGES_REQUESTED** [review]

**Comment by techcobweb** [comment]
Can you please associate this PR with the story it's contributing to ? We need to all get better at forming these links so they can be followed if people want to navigate from the story to the code...

File: pkg/runs/runsDownload.go (hunk 1 above)
Comment (Wyvinar) [reply pkg/runs/runsDownload.go]: Made the error returning from the embedded function more readable to remove this everywhere.

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-cli-pr-197-5874z for details. If you are unable to do so, please contact a member of the Galasa team.

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-cli-pr-197-5874z-r-t5thp for details. If you are unable to do so, please contact a member of the Galasa team.

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-cli-pr-197-5874z-r-lzcmd for details. If you are unable to do so, please contact a member of the Galasa team.

**Comment by galasa-team** [comment]
Build successful

**Review by KirbyKatcher — APPROVED** [review]
looks good

**Pull request merged**